keyboards/crkbd/keymaps/vayashiko/keymap.c

```diff
@@ -0,0 +1,249 @@
+      KC_ALTKN,   KC_GRV, KC_XXXXX, KC_XXXXX,  KC_MHEN,     KC_G,                  KC_XXXXX,  KC_HENK,  JP_LBRC,  JP_RBRC, KC_XXXXX,  KC_JYEN,\
+  //|---------+---------+---------+---------+---------+---------|                |---------+---------+---------+---------+---------+---------|
+       KC_LSFT, KC_XXXXX, KC_XXXXX, KC_XXXXX, KC_XXXXX, KC_XXXXX,                  JP_EQL,JP_AT,LSFT(JP_LBRC),LSFT(JP_RBRC), KC_XXXXX,  JP_UNDS,\
+  //|---------+---------+---------+---------+---------+---------+------|  |------+---------+---------+---------+---------+---------+---------|
+                                              KC_CTLTB, KC_LOWER,KC_SPC,   KC_ENT, KC_RAISE, KC_ALTKN \
+                                          //`--------------------------'  `--------------------------'
+  ),
+
+  [_ADJUST] = LAYOUT( \
+  //,-----------------------------------------------------------.                ,-----------------------------------------------------------.
+        KC_ESC,  KC_LRST, KC_XXXXX, KC_XXXXX,     KC_R, KC_XXXXX,                  KC_XXXXX, KC_XXXXX,    KC_UP, KC_XXXXX, KC_XXXXX, KC_XXXXX,\
+  //|---------+---------+---------+---------+---------+---------|                |---------+---------+---------+---------+---------+---------|
+      KC_ALTKN,  KC_HOME,     KC_S, KC_XXXXX,   KC_END,     KC_G,                  KC_XXXXX,  KC_LEFT,  KC_DOWN, KC_RIGHT, KC_XXXXX, KC_XXXXX,\
+  //|---------+---------+---------+---------+---------+---------|                |---------+---------+---------+---------+---------+---------|
+       KC_LSFT,  KC_LHUD,  KC_LSAD,  KC_LVAD, KC_XXXXX, KC_XXXXX,                  KC_XXXXX, KC_XXXXX,  KC_DOWN, KC_XXXXX, KC_XXXXX,  KC_RSFT,\
+  //|---------+---------+---------+---------+---------+---------+------|  |------+---------+---------+---------+---------+---------+---------|
+                                              KC_GUIEI, KC_LOWER,KC_SPC,   KC_ENT, KC_RAISE, KC_ALTKN \
+                                          //`--------------------------'  `--------------------------'
+  )
+};
+
+int RGB_current_mode;
+
+void persistent_default_layer_set(uint16_t default_layer) {
+  eeconfig_update_default_layer(default_layer);
+  default_layer_set(default_layer);
+}
+
+// Setting ADJUST layer RGB back to default
+void update_tri_layer_RGB(uint8_t layer1, uint8_t layer2, uint8_t layer3) {
+  if (IS_LAYER_ON(layer1) && IS_LAYER_ON(layer2)) {
+    layer_on(layer3);
+  } else {
+    layer_off(layer3);
+  }
+}
+
+void matrix_init_user(void) {
+    #ifdef RGBLIGHT_ENABLE
+      RGB_current_mode = rgblight_config.mode;
+    #endif
+    //SSD1306 OLED init, make sure to add #define SSD1306OLED in config.h
+    #ifdef SSD1306OLED
+        iota_gfx_init(!has_usb());   // turns on the display
+    #endif
+}
+
+//SSD1306 OLED update loop, make sure to add #define SSD1306OLED in config.h
+#ifdef SSD1306OLED
```

Comment:
the `SSD1306OLED` code has been deprecated, and the "OLED Driver" feature should be used instead. 

You can take a look at the default keymap for updates, or use the conversion guide:
https://docs.qmk.fm/#/feature_oled_driver?id=ssd1306h-driver-conversion-guide